### PR TITLE
Translation for access-authn-authz to Bahasa Indonesia

### PR DESCRIPTION
**Pull request by imrenagi**
Adding translation to Bahasa Indonesia for `/docs/reference/access-authn-authz/rbac/` 

Signed-off-by: Imre Nagi <imre.nagi2812@gmail.com>

### CODE REVIEW

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -0,0 +1,1195 @@
...
+rules:
+- apiGroups: [""]
+  #
+  # pada tingkat HTTP, nama dari sumber daya untuk mengakses objek Pod
+  # adalah "pods"
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+```
+
+Mengizinkan pembacaan/penulisan Deployments (pada tingkat HTTP: objek dengan `"deployments"`
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:368]:
```suggestion
Mengizinkan pembacaan/penulisan Deployment (pada tingkat HTTP: objek dengan `"deployments"` 
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 2]
```diff
@@ -0,0 +1,1195 @@
...
+```yaml
+rules:
+- nonResourceURLs: ["/healthz", "/healthz/*"] # '*' in a nonResourceURL is a suffix glob match
+  verbs: ["get", "post"]
+```
+
+### Mengacu Pada Subjek
+
+RoleBinding atau ClusterRoleBinding mengikat sebuah peran ke subjek.
+Subjek dapat berupa kelompok, pengguna atau ServiceAccounts.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:440]: Replace all ServiceAccounts in this page with ServiceAccount
```suggestion
Subjek dapat berupa kelompok, pengguna atau ServiceAccount.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 3]
```diff
@@ -0,0 +1,1195 @@
...
+
+Kubernetes merepresentasikan _usernames_ sebagai string.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:442]:
```suggestion
Kubernetes merepresentasikan _username_ sebagai string.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 4]
```diff
@@ -0,0 +1,1195 @@
...
+Ini bisa berupa: nama sederhana, seperti "alice"; email, seperti "bob@example.com";
+atau ID pengguna numerik yang direpresentasikan sebagai string. Terserah kamu sebagai administrator klaster
+untuk mengkonfigurasi [authentication modules](/docs/reference/access-authn-authz/authentication/)
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:445]: can be translated as well
```suggestion
untuk mengkonfigurasi [modul otentikasi](/docs/reference/access-authn-authz/authentication/)
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 5]
```diff
@@ -0,0 +1,1195 @@
...
+sehingga otentikasi menghasilkan _usernames_ dalam format yang kamu inginkan.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:446]:
```suggestion
sehingga otentikasi menghasilkan _username_ dalam format yang kamu inginkan.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 6]
```diff
@@ -0,0 +1,1195 @@
...
+bahwa kamu tidak memiliki pengguna atau grup dengan nama yang dimulai dengan `system:` secara tidak sengaja.
+Selain awalan khusus ini, sistem otorisasi RBAC tidak memerlukan format apa pun
+untuk nama pengguna.
+{{< /caution >}}
+
+Di Kubernetes, modul otentikasi menyediakan informasi grup.
+Grup, seperti halnya pengguna, direpresentasikan sebagai string, dan string tersebut tidak memiliki format tertentu,
+selain awalan `system:` yang sudah direservasi.
+
+[ServiceAccounts](/docs/tasks/configure-pod-container/configure-service-account/) memiliki nama yang diawali dengan `system:serviceaccount:`, dan menjadi milik grup yang diawali dengan nama `system:serviceaccounts:`.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:459]:
```suggestion
[ServiceAccount](/docs/tasks/configure-pod-container/configure-service-account/) memiliki nama yang diawali dengan `system:serviceaccount:`, dan menjadi milik grup yang diawali dengan nama `system:serviceaccounts:`.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 7]
```diff
@@ -0,0 +1,1195 @@
...
+
+{{< note >}}
+- `system:serviceaccount:` (tunggal) adalah awalan untuk _service account usernames_.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:462]: should be written with CamelCase
```suggestion
- `system:serviceaccount:` (tunggal) adalah awalan untuk ServiceAccount _username_.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 8]
```diff
@@ -0,0 +1,1195 @@
...
+- `system:serviceaccounts:` (jamak) adalah awalan untuk _service account_ grup.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:463]:
```suggestion
- `system:serviceaccounts:` (jamak) adalah awalan untuk ServiceAccount grup.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 9]
```diff
@@ -0,0 +1,1195 @@
...
+{{< /note >}}
+
+#### Contoh RoleBinding {#role-binding-examples}
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:466]: you can remove the custom anchor link here, and refer with `#contoh-rolebinding`
```suggestion
#### Contoh RoleBinding
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 10]
```diff
@@ -0,0 +1,1195 @@
...
+
+Contoh-contoh berikut ini hanya potongan `RoleBinding` yang hanya memperlihatkan
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:468]: no need for backtick
```suggestion
Contoh-contoh berikut ini hanya potongan RoleBinding yang hanya memperlihatkan
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 11]
```diff
@@ -0,0 +1,1195 @@
...
+Untuk grup bernama `frontend-admins`:
+
+```yaml
+subjects:
+- kind: Group
+  name: "frontend-admins"
+  apiGroup: rbac.authorization.k8s.io
+```
+
+Untuk _service account_ bawaan di _namespace_ "kube-system":
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:489]:
```suggestion
Untuk ServiceAccount bawaan di _namespace_ "kube-system":
```

Comment (girikuncoro) [reply content/id/docs/reference/access-authn-authz/rbac.md:489]: let's replace all _service account_ in this page with ServiceAccount

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 12]
```diff
@@ -0,0 +1,1195 @@
...
+
+## Role dan RoleBindings Bawaan
+
+API membuat satu set objek ClusterRole dan ClusterRoleBinding bawaan. 
+Sebagian besar dari objek berawalan `system:`, menunjukkan bahwa sumber daya tersebut
+secara langsung dikelolah oleh _control plane_ klaster. Seluruh ClusterRole dan ClusterRoleBinding dilabeli dengan
+`kubernetes.io/bootstrapping=rbac-defaults`.
+
+{{< caution >}}
+Berhati-hatilah saat memodifikasih CLusterRole dan ClusterRoleBinding dengan nama yang
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:554]: typo
```suggestion
Berhati-hatilah saat memodifikasi ClusterRole dan ClusterRoleBinding dengan nama yang
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 13]
```diff
@@ -0,0 +1,1195 @@
...
+<td>Mengizinkan akses baca pada _API discovery endpoints_ yang dibutuhkan untuk menemukan dan melakukan negosiasi pada tingkat API. Sebelum v1.14, peran ini juga terikat pada <tt>system:unauthenticated</tt> secara bawaan.</td>
+</tr>
+<tr>
+<td><b>system:public-info-viewer</b></td>
+<td><b>system:authenticated</b> and <b>system:unauthenticated</b> groups</td>
+<td>Mengizinkan akses baca pada informasi yang tidak sensitif tentang klaster. Diperkenalkan pada Kubernetes v1.14.</td>
+</tr>
+</table>
+
+### Peran Pengguna
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:614]: Role shouldn't get translated
```suggestion
### Role Pengguna
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 14]
```diff
@@ -0,0 +1,1195 @@
...
+
+Beberapa ClusterRole bawaan tidak diawali dengan `system:`. Ini dimaksudkan untuk peran pengguna.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:616]:
```suggestion
Beberapa ClusterRole bawaan tidak diawali dengan `system:`. Ini dimaksudkan untuk Role pengguna.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 15]
```diff
@@ -0,0 +1,1195 @@
...
+Ini termasuk peran super-user (`cluster-admin`), peran yang dimaksudkan untuk diberikan akses seluruh klaster dengan
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:617]:
```suggestion
Ini termasuk peran Role super-user (`cluster-admin`), yang dimaksudkan untuk diberikan akses seluruh klaster dengan
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 16]
```diff
@@ -0,0 +1,1195 @@
...
+menggunakan ClusterRoleBinding, dan peran yang dimaksudkan untuk diberikan pada namespace tertentu
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:618]:
```suggestion
menggunakan ClusterRoleBinding, dan Role yang dimaksudkan untuk diberikan pada Namespace tertentu 
```

Comment (girikuncoro) [reply content/id/docs/reference/access-authn-authz/rbac.md:618]: should replace namespace in this page entirely with Namespace

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 17]
```diff
@@ -0,0 +1,1195 @@
...
+Namun, peran ini memungkinkan untuk mengakses secret dan menjalankan pod seperti ServiceAccount dalam namespace, 
+sehingga dapat digunakan untuk mendapatkan tingkat akses API dari setiap ServiceAccount di namespace. 
+</td>
+</tr>
+<tr>
+<td><b>view</b></td>
+<td>None</td>
+<td>Mengizinkan akses baca untuk melihat hampir seluruh objek dalam namespace.
+
+Ini tidak memungkinkan untuk melihat peran dan RoleBinding.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:669]:
```suggestion
Ini tidak memungkinkan untuk melihat Role dan RoleBinding.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 18]
```diff
@@ -0,0 +1,1195 @@
...
+
+Peran ini tidak memungkikan melihat Secret, karena pembacaan konten Secret memungkinkan
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:671]:
```suggestion
Role ini tidak memungkikan melihat Secret, karena pembacaan konten Secret memungkinkan
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 19]
```diff
@@ -0,0 +1,1195 @@
...
+Jika digunakan dalam <b>RoleBinding</b>, ini memungkikan akses baca/tulis ke sebagian besar sumber daya di sebuah namespace,
+termasuk kemampuan untuk membuat Role dan RoleBinding dalam namespace. 
+Peran ini tidak memungkinkan akses tulis pada kuota sumber daya atau ke namespace itu sendiri.</td>
+</tr>
+<tr>
+<td><b>edit</b></td>
+<td>None</td>
+<td>Mengizinkan akses baca/tulis pada seluruh objek dalam namespace.
+
+Peran ini tidak memungkinkan untuk melihat dan merubah Role dan RoleBinding.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:659]:
```suggestion
Role ini tidak memungkinkan untuk melihat dan merubah Role dan RoleBinding.
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 20]
```diff
@@ -0,0 +1,1195 @@
...
+Namun, peran ini memungkinkan untuk mengakses secret dan menjalankan pod seperti ServiceAccount dalam namespace,
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:660]:
```suggestion
Namun, Role ini memungkinkan untuk mengakses Secret dan menjalankan Pod seperti ServiceAccount dalam Namespace, 
```

File: content/id/docs/reference/access-authn-authz/rbac.md [hunk 21]
```diff
@@ -0,0 +1,1195 @@
...
+
+Peran ini tidak memungkinkan untuk melihat dan merubah Role dan RoleBinding.
+Namun, peran ini memungkinkan untuk mengakses secret dan menjalankan pod seperti ServiceAccount dalam namespace, 
+sehingga dapat digunakan untuk mendapatkan tingkat akses API dari setiap ServiceAccount di namespace. 
+</td>
+</tr>
+<tr>
+<td><b>view</b></td>
+<td>None</td>
+<td>Mengizinkan akses baca untuk melihat hampir seluruh objek dalam namespace.
```
Comment (girikuncoro) [inline content/id/docs/reference/access-authn-authz/rbac.md:667]:
```suggestion
<td>Mengizinkan akses baca untuk melihat hampir seluruh objek dalam Namespace.
```

**Comment by imrenagi** [comment]
gan @girikuncoro I've updated the PR with your suggestions. Please check and thanks so much for the help.

**Pull request merged**